### ISSUE TITLE
RC test: pH Scale: Basics 1.2.0-rc.1 (aborted)

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/tasks#323]
Related to #322...

Per @ariel-phet, I need to publish this and create the test matrix. Since it's a subset of 'pH Scale', we'll be doing spot testing, and publishing 'pH Scale' and 'pH Scale: Basics' concurrently.

**Comment by pixelzoom** [comment]
RC version:
http://www.colorado.edu/physics/phet/dev/html/ph-scale-basics/1.2.0-rc.1/ph-scale-basics_en.html

Test matrix:
https://docs.google.com/spreadsheets/d/1F0JBJ2MyPkKk4LmFSs224OpPKUNR8lQpUPOeOw56uqk/edit#gid=1465534985

*This sim only needs to be spot tested.*  This sim is the "Macro" screen from ph-scale, and ph-scale is being RC tested in #323. The only behavior that's different is that the beaker does NOT automatically fills when the sim starts or is reset.

**Comment by pixelzoom** [comment]
Please halt RC testing. This is serious: https://github.com/phetsims/scenery/issues/455#issuecomment-133221684

**Comment by pixelzoom** [comment]
phetsims/scenery#455 was fixed and 1.2.0-rc.1 has been republished. Please resume testing.

**Comment by ariel-phet** [comment]
@elisemorgan @phet-steele @bryo5363 @orejolaphet please test next in line after #322

**Comment by ariel-phet** [comment]
And ideally for efficiency test concurrently (since only needs spot testing) on each platform

**Comment by pixelzoom** [comment]
This test task is on hold until we've done more beta testing of Rosetta with translators. Assigning to me for now. I'll pass it back to @ariel-phet when I feel comfortable proceeding.

**Comment by pixelzoom** [comment]
I deleted 1.2.0-rc.1 from spot, will re-publish it when we resume RC testing.

**Comment by pixelzoom** [comment]
Unassigning until we resume RC testing.

**Comment by pixelzoom** [comment]
@ariel-phet said that ph-scale-basics may not be first on the redeploy schedule. Assigning this back to him.  Assign back to me with instructions when ready.

**Comment by pixelzoom** [comment]
Since we'll be starting over on RC testing, I deleted the '1.2' branch of ph-scale-basics.

**Comment by pixelzoom** [comment]
Since this RC test was aborted awhile back, I'm going to close this issue and start fresh, so that we don't have any confusion.
